Fail to build project following README

##  Summary
Failed to build project while follow the master branch readme

## Version
latest master

## Additional context
I follow the readme:
```
$ git clone https://github.com/{your-personal-repo}/tui.editor.git
$ npm install
$ cd apps/editor
$ npm install
```

But when i run 
`npm run serve`
i have the following error:
```
   
    ERROR in ./src/js/convertor.js
    Module not found: Error: Can't resolve '@toast-ui/to-mark' in '/xxxx/tui.editor/apps/editor/src/js'
     @ ./src/js/convertor.js 5:0-39 164:19-25
     @ ./src/js/editor.js
     @ ./src/js/index.js
    
    ERROR in ./src/js/toMarkRenderer.js
    Module not found: Error: Can't resolve '@toast-ui/to-mark' in '/xxxx/tui.editor/apps/editor/src/js'
     @ ./src/js/toMarkRenderer.js 1:0-39 42:15-21 42:39-45
     @ ./src/js/editor.js
     @ ./src/js/index.js
    
    ERROR in ./src/js/editor.js
    Module not found: Error: Can't resolve '@toast-ui/toastmark' in '/xxxx/tui.editor/apps/editor/src/js'
     @ ./src/js/editor.js 75:0-48 182:25-34
     @ ./src/js/index.js
    
    ERROR in ./src/js/markdownToHTML.js
    Module not found: Error: Can't resolve '@toast-ui/toastmark' in '/xxxx/tui.editor/apps/editor/src/js'
     @ ./src/js/markdownToHTML.js 1:0-63 6:19-25 10:19-35
     @ ./src/js/editor.js
     @ ./src/js/index.js
    
    ERROR in ./src/js/mdPreview.js
    Module not found: Error: Can't resolve '@toast-ui/toastmark' in '/xxxx/tui.editor/apps/editor/src/js'
     @ ./src/js/mdPreview.js 11:0-55 40:23-39
     @ ./src/js/editor.js
     @ ./src/js/index.js
    
    ERROR in ./src/js/convertor.js
    Module not found: Error: Can't resolve '@toast-ui/toastmark' in '/xxxx/tui.editor/apps/editor/src/js'
     @ ./src/js/convertor.js 6:0-63 36:24-30 40:22-38
     @ ./src/js/editor.js
     @ ./src/js/index.js
```
(xxxx is my local folder path)
Any extra step needed to build the project?

--- a/README.md
+++ b/README.md
@@ -125,12 +125,13 @@ TOAST UI products are open source, so you can create a pull request(PR) after yo
 
 ### Setup
 
-Fork `master` branch into your personal repository. Clone it to local computer. Install node modules. Before starting development, you should check to haveany errors.
+Fork `master` branch into your personal repository. Clone it to local computer. Install node modules. Before starting development, you should check to see if there are any errors.
 
 ```sh
 $ git clone https://github.com/{your-personal-repo}/tui.editor.git
 $ cd [project-name]
 $ npm install
+$ npm run setup:libs
 $ cd [project-name]/apps/editor
 $ npm install
 $ npm run test

--- a/package.json
+++ b/package.json
@@ -20,6 +20,12 @@
     "build:uml": "lerna run --scope @toast-ui/editor-plugin-uml build",
     "build:toastmark": "lerna run --scope @toast-ui/toastmark build",
     "build:squire": "lerna run --scope @toast-ui/squire build",
-    "build:to-mark": "lerna run --scope @toast-ui/to-mark build"
+    "build:to-mark": "lerna run --scope @toast-ui/to-mark build",
+    "install:toastmark": "cd libs/toastmark && npm install",
+    "install:squire": "cd libs/squire && npm install",
+    "install:to-mark": "cd libs/to-mark && npm install",
+    "install:libs": "npm run install:toastmark && npm run install:squire && npm run install:to-mark",
+    "build:libs": "npm run build:toastmark && npm run build:squire && npm run build:to-mark",
+    "setup:libs": "npm run install:libs && npm run build:libs"
   }
 }

